tests/test_lldp.py

```diff
@@ -106,11 +106,36 @@ def test_subtype_lldp_loc_man_addr_table(self):
 
 
     def test_subtype_lldp_rem_man_addr_table(self):
+        # Verfiy both valid ipv4 and ipv6 address exist
         for entry in range(3, 6):
             mib_entry = self.lut[(1, 0, 8802, 1, 1, 2, 1, 4, 2, 1, entry)]
             ret = mib_entry(sub_id=(1, 1))
             self.assertIsNotNone(ret)
             print(ret)
+        # Verfiy valid ipv4 and invalid ipv6 address exist
+        for entry in range(3, 6):
+            mib_entry = self.lut[(1, 0, 8802, 1, 1, 2, 1, 4, 2, 1, entry)]
+            ret = mib_entry(sub_id=(1, 5))
+            self.assertIsNotNone(ret)
```

Comment:
>ret [](start = 33, length = 3)

Could you test the ip address representation in these test cases?
`print` is not necessary for unit test.

Reply:
@qiluo-msft Updated to verify both v4 and v6 representation.